examples/lines_bars_and_markers/hat_graph.py

```diff
@@ -0,0 +1,60 @@
+"""
+===============================
+Hat Graph with labels
+===============================
+This example shows a how to create a hat graph and how to annotate with labels.
```

Comment:
Maybe add a reference to the article?   
```suggestion
This example shows a how to create a hat graph (https://doi.org/10.1186/s41235-019-0182-3) 
and how to annotate with labels.
```

Reply:
Yeh Are we allowed to add references didnt no if it is allowed